test/test-test262.js

```diff
@@ -138,7 +138,7 @@ function hasExcludedInfo(testInfo) {
       // 'References observe the initialization of lexical bindings', // https://github.com/Agoric/make-importer/issues/40
       // 'References observe the mutation of initialized bindings', // https://github.com/Agoric/make-importer/issues/40
 
-      'Requested modules are evaluated prior to the requesting module in source code order', // https://github.com/Agoric/make-importer/issues/43
+      // 'Requested modules are evaluated prior to the requesting module in source code order', // https://github.com/Agoric/make-importer/issues/43
```

Comment:
why is this commented out?

Reply:
It's a test exclusion list.  Commenting it out allows the test to run.